x-pack/elastic-agent/docs/fleet-server-bootstrap.asciidoc

```diff
@@ -0,0 +1,83 @@
+[[fleet-server-bootstrap]]
+== Fleet Server Bootstrap
+
+Elastic Agent with Fleet Server has a bootstrap process that it uses to get
+Fleet Server up and running under Elastic Agent.
+
+Upon `install` or `enroll` with Elastic Agent will bootstrap a Fleet Server
+when the `--fleet-server-es` command-line option is provided. In this mode
+Elastic Agent will only communicate with its local Fleet Server and expose
+the Fleet Server over the `:8220` port.
```

Comment:
Based on this: https://github.com/elastic/fleet-server/pull/967/files
Both 8220 and 8221 will be used. 8221 for exclusive communication with the local Elastic Agent running.

Reply:
I updated to documentation to use :8221 and added description on the difference between :8220 and :8221.